solidity/contracts/deposit/Deposit.sol

```diff
@@ -122,7 +122,7 @@ contract Deposit is DepositFactoryAuthority {
         self.tbtcToken = _TBTCToken;
         self.tbtcDepositToken = _TBTCDepositToken;
         self.feeRebateToken = _FeeRebateToken;
-        self.VendingMachine = _VendingMachine;
+        self.VendingMachineAddress = _VendingMachineAddress;
```

Comment:
… Feels like all of these should start with a lowercase? And does that maybe also include all the `_feeRebateToken`, `_tbtcDepositToken`, etc?

Reply:
I can go either way, as long as we're consistent. 
Changing to lowercase for consistency